pkg/virt-handler/vm.go

```diff
@@ -1704,8 +1700,8 @@ func (c *VirtualMachineController) Run(threadiness int, stopCh chan struct{}) {
 		go wait.Until(c.runWorker, time.Second, stopCh)
 	}
 
-	<-stopCh
 	<-heartBeatDone
+	<-stopCh
```

Comment:
nit: why is this needed? I'm pretty sure that the behavior is equivalent to before your changes.

Reply:
Yes, the behavior is equivalent.
I do that to highlight that the heartbeat should succeed before moving on. :)
Thank you!
I will rollback if you think it's misleading